### ISSUE TITLE
Fill buckets in chests

### ISSUE THREAD

**Issue by DanHolli** [shadowfacts/Funnels#2]
a funnel pointing at a chest that has buckets in it will fill them when possible
also a vacuum funnel which sucks up a liquid below it like lava in actually additions lava creator multi block in which the lava in only accessible by the top

**Comment by shadowfacts** [comment]
Chests: Maybe

Vacuum funnels: No, that's not how funnels work. There are other mods that add pumps that you can use.

**Comment by DanHolli** [comment]
I don't think it would be easy as you would need to unstack buckets if needed and fill them
but it shouldn't be too hard as you created Tesla
